src/Parser/Mage/Frost/Modules/Features/BrainFreeze.js

```diff
@@ -11,81 +11,104 @@ const debug = false;
 // Brain Freeze appears to always fall after Flurry cast, but not always on same timestamp. Giving a margin here.
 const PROC_WINDOW_MS = 100;
 
+// When Glacial Spike is talented, it's OK to overwrite Brain Freeze procs if you have at least this many icicles during the frost bolt cast
```

Comment:
I talked to the theorycrafters in the Mage Discord and as it turns out, the breakpoint is actually 4 not 3. This is likely a confusion on my part as during prepatch they told me to change it to 4 and I did ... and then there were complaints last week in the Mage Discord about how wowanalyzer contradicted Altered Time and Altered Time said 3. So I assumed Altered Time was right and changed it back to 3 so it matched.

But as it turns out, Altered Time was wrong and it should be 4.

Also confirmed that if they get Brain Freeze on the 3rd Frostbolt and are in the middle of casting Frostbolt, then they should hold Brain Freeze cause the current Frostbolt would give them 4.

![image](https://user-images.githubusercontent.com/1304554/45602265-ff30c800-b9e0-11e8-88e2-eb7be0503948.png)